packages/prover/package.json

```diff
@@ -53,9 +53,12 @@
     "lint:fix": "yarn run lint --fix",
     "pretest": "yarn run check-types",
     "test": "yarn test:unit && yarn test:e2e",
-    "test:unit": "nyc --cache-dir .nyc_output/.cache -e .ts mocha 'test/unit/**/*.test.ts'",
-    "test:browsers": "yarn karma start karma.config.cjs",
-    "test:e2e": "LODESTAR_PRESET=minimal mocha 'test/e2e/**/*.test.ts'",
+    "test:unit": "vitest --run --dir test/unit/ --coverage",
+    "test:browsers": "test:browsers:chrome && test:browsers:firefox && test:browsers:electron",
+    "test:browsers:chrome": "vitest --run --browser chrome --config ./vitest.browser.config.ts --dir test/unit",
+    "test:browsers:firefox": "vitest --run --browser firefox --config ./vitest.browser.config.ts --dir test/unit",
+    "test:browsers:electron": "echo 'Electron tests will be introduced back in the future as soon vitest supports electron.'",
```

Comment:
We can bring back electron to our stack, soon when `vitest` bring it support, else writing a custom [BrowserProvider](https://vitest.dev/config/#browser-provider) will not be difficult for it.